src/notificationcenter.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace XamCore.NotificationCenter {
 #if XAMCORE_2_0 || !MONOMAC
-	[iOS (8,0)][Mac (10,10)]
+	[iOS (8,0)][Mac (10,10, onlyOn64 : true)]
```

Comment:
The 32-bit test additions and the corresponding fixes are unrelated can go in a different PR.